packages/botonic-react/src/constants.js

```diff
@@ -49,12 +49,16 @@ export const WEBCHAT = {
     botMessageImage: 'message.bot.image',
     botMessageImageStyle: 'message.bot.imageStyle',
     botMessageBlobTick: 'message.bot.blobTick',
+    botMessageBlobTickSize: 'message.bot.blobTickStyle',
     botMessageStyle: 'message.bot.style',
     botMessageBackground: 'message.bot.style.background',
+    botMessageBorderColor: 'message.bot.style.borderColor',
     botMessageBlobWidth: 'message.bot.blobWidth',
     userMessageStyle: 'message.user.style',
     userMessageBlobTick: 'message.user.blobTick',
+    userMessageBlobTickSize: 'message.user.blobTickStyle',
     userMessageBackground: 'message.user.style.background',
+    userMessageBorderColor: 'message.user.style.borderColor',
```

Comment:
`botMessageBlobTickSize` and `userMessageBlobTickSize` are not consistent with the naming here. Can you please rename it as `botMessageBlobTickStyle` and  `userMessageBlobTickStyle`